src/algo/floyd_warshall.rs

```diff
@@ -10,7 +10,7 @@ use crate::visit::{
 #[allow(clippy::type_complexity, clippy::needless_range_loop)]
 /// \[Generic\] [Floyd–Warshall algorithm](https://en.wikipedia.org/wiki/Floyd%E2%80%93Warshall_algorithm) is an algorithm for all pairs shortest path problem
 ///
-/// Compute shortest paths in a weighted graph with positive or negative edge weights (but with no negative cycles)
+/// Compute distance of shortest paths in a weighted graph with positive or negative edge weights (but with no negative cycles)
```

Comment:
"Length" makes more sense here.
|
```suggestion
/// Compute the length of each shortest path in a weighted graph with positive or negative edge weights (but with no negative cycles).
```